vimeo-networking/src/main/java/com/vimeo/networking/model/User.java

```diff
@@ -443,7 +447,7 @@ public String getId() {
      */
     @Nullable
     public Boolean getIsStaff() {
-        return mIsStaff;
+        return getBadge() != null && getBadge().getBadgeType() == UserBadgeType.STAFF;
```

Comment:
Remove the `@Nullable` and make the return type `boolean`.

Reply:
Fixed in https://github.com/vimeo/vimeo-networking-java/pull/410/commits/3f30862b439cb201f4986865d4347cd968db7770